addon/i3dio/utility.py

```diff
@@ -82,10 +82,17 @@ def as_fs_relative_path(filepath: str):
 def sort_blender_objects_by_name(objects: List[BlenderObject]) -> List[BlenderObject]:
     return sorted(objects, key=lambda x: x.name)
 
+
 """
 Blenders outliner does not follow a stricly lexographical ordering, but rather what is called a "natural" ordering.
 This function implements the same ordering as per https://github.com/blender/blender/blob/b0e7a6db56caf6669b6fade1622710d70b96483e/source/blender/blenlib/intern/string.c#L727,
 with the use of a regex as detailed in this answer on stackoverflow https://stackoverflow.com/a/16090640
 """
+
+
 def sort_blender_objects_by_outliner_ordering(objects: List[BlenderObject]) -> List[BlenderObject]:
     return sorted(objects, key=lambda s: [int(t) if t.isdigit() else t.lower() for t in re.split('(\d+)', s.name)])
+
+
+def blender_version() -> int:

```

Comment:
The name of this function is a bit misleading since you aren't returning the version, but just the version Major. Also why does this need a function to begin with, can't you just use `bpy.app.version` directly?